### ISSUE TITLE
Journey Map unable to download

### ISSUE THREAD

**Issue by redwolf41** [WTFGeeks/Natural-Magic#266]
Hello! I was trying to install the 0.8.1 version with Journey Map, VeinMiner and ExtrabiomesXL added, and it couldnt install JourneyMap. The website it tried to pull up had nothing there. [This ](https://minecraft.curseforge.com/mc-mods/32274-journeymap-32274/files/2221283/download) is the site it went to. I tried to find the file the pop up was referring to but was unable to find it.

**Comment by rabidfraggle** [comment]
I am currently running 0.8.1 and did not have a problem installing Journey Map.

Are you using the latest version of ATLauncher?

**Comment by redwolf41** [comment]
Yup. ATLauncher checks for and installs any updates every time I open it.

**Comment by rabidfraggle** [comment]
![image](https://cloud.githubusercontent.com/assets/25734693/23031821/56e8d12c-f440-11e6-8813-bd6a5fe33840.png)

That is strange, I just tried to re-install an instance and got the error.

**Comment by rabidfraggle** [comment]
Looking at the  mod file list:
https://minecraft.curseforge.com/projects/journeymap-32274/files?sort=game-version
I do not see a 1.8.1 version.. only 1.8

I do not see the file referenced available for download.
Looking at the current files available :"journeymap-1.8-5.1.3-unlimited.jar"
It looks like they changed the naming structure, to minecraft version - mod version
instead of mod version - minecraft version. 

Have you tried to manually download a different version and add it locally?

**Comment by redwolf41** [comment]
I did, but that just gave me a forge error once it loaded. I tried grabbing the most up to date forget file, but that didn't fix it either.

**Comment by Hunter4618** [comment]
Did anyone get it to work because I'm also trying to download it and it's coming up with the same error

**Comment by Hunter4618** [comment]
when I opened the mod folder it also said all the files were Ineligible and I don't know what that means

**Comment by redwolf41** [comment]
I gave up after a couple tries. I still have it downloaded on my old laptop. When I get home I'll boot it up and see if the old download for it still works or not.

**Comment by Hunter4618** [comment]
thanks I downloaded it without journeymap because I still want to play it but having the map and the waypoints is very convenient

**Comment by redwolf41** [comment]
Seriously. I have a hard time without the waypoints since I travel so far. TT-TT

**Comment by Hunter4618** [comment]
ya Ik being able to teleport is SO MUCH easier

**Comment by Hunter4618** [comment]
but do you have any idea why the launcher doesn't recognize when I downloaded the mod?

**Comment by Hunter4618** [comment]
I play mods a lot but I don't actually know how they work that much :P

**Comment by redwolf41** [comment]
The problem is that the file that was used when downloading Natural Magic is missing, even when it kicks you to the website. Thats why I want to test an old install that has Journey Map and see if that works.

**Comment by Hunter4618** [comment]
o ok

**Comment by Hunter4618** [comment]
thanks

**Comment by pookalion** [comment]
I'm having this same problem.  I tried manually downloading journeymap and re-naming it in my folder to the same thing as it's looking for, but it didn't work.

**Comment by redwolf41** [comment]
The install on my old computer works fine. I'm gonna grab the journeymap file and move it to my new computer so I can still use it.

**Comment by Hunter4618** [comment]
So did you use an older version of Natural magic to find the right file?

**Comment by Hunter4618** [comment]
And do you know if I would be able to find it on the interweb anywhere?

**Comment by redwolf41** [comment]
No its just an old computer. The version of natural magic is up to date. As for finding it online, I don't know to be honest.

**Comment by Hunter4618** [comment]
I'm trying to reinstall Natural magic right now

**Comment by Hunter4618** [comment]
I tried to install voxelmap instead but it still doesn't seem to be working

**Comment by Hunter4618** [comment]
Ok I managed to install voxelmap (forgot i had to run liteloader too :P) and its working so i'm good

**Comment by RikeSilvermist** [comment]
I think what the problem is the fact that you are installing the wrong journey map file. I had this problem a while back and felt rather stupid about it, there is a download the latest file near the top in green but that isnt the one you want. You want the one about a quarter of the was down the page that is grey.

**Comment by sniper9591** [comment]
@RikeSilvermist Do you remember the name/version number of the file that worked? None of the links were grey when I looked through the version list, and I tried all of the files labelled as being for 1.7.10, which is the version that Natural Magic was written for, but none of them worked.
